### DISCUSSION TITLE
Stuck at 'Build finished'

### DISCUSSION THREAD

**Discussion by marcustut** [antfu/vite-ssg#82] [Q&A]
I'm using [Vitesse](https://github.com/antfu/vitesse)

but vite-ssg is not working during `build`, the original version of vite-ssg is v0.12.0, I tried upgrading to v0.14.5 and the build step still doesn't work.

It will stop at Build finished but will never exit, therefore when deploying the project to Netlify, it will not be successful because the build doesn't exit.

I tried using `vite build` instead of `vite-ssg build` and vite build works perfectly fine

![image](https://user-images.githubusercontent.com/59773847/127761466-99e81482-4343-461c-a3fe-58be23e1700f.png)

**Comment by antfu** [comment]
From what I can see, it seems like @headlessui/vue is not SSR compatible.

**Reply by marcustut** [reply]
Thanks for pointing that out, never thought of that since headlessui works perfectly fine with SSR in react. First time using it for a vue project.

Further inspection at the issue leads me to this [issue](https://github.com/tailwindlabs/headlessui/issues/614) where it does mention that 'Select' and 'Dialog' components of headlessui doesn't support SSR yet.

That said, I think exiting the build step with an error is more verbose than remaining stuck at 'Build finished'
